src/5.0_to_5.1/migrateSettings.test.ts

```diff
@@ -0,0 +1,180 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+import _cloneDeep from "lodash/cloneDeep";
+import { migrateSettings } from "./migrateSettings";
+import { contentServer as testContentServer } from "./__test_resources__/contentServer";
+
+describe("migrateSettings", () => {
+  let contentServer: ContentRecord;
+
+  beforeEach(() => {
+    contentServer = _cloneDeep(testContentServer);
+  });
+
+  it("doesn't do anything if there is no settings", () => {
+    const contentServerClone = _cloneDeep(contentServer);
```

Comment:
Why clone the content server a second time ?
We already have `testContentServer` and `contentServer`